src/python/dxpy/bindings/apollo/assay_filtering_json_schemas.py

```diff
@@ -9,7 +9,7 @@
     },
     "expression": {
         "type": dict,
-        "properties": {"min_value": {"type": str}, "max_value": {"type": str}},
+        "properties": {"min_value": {"type": str,"required":False}, "max_value": {"type": str,"required":False  }},
```

Comment:
Was the class not working as expected without `"required": False`? If so, this is probably a bug and I can look into it and fix it.